src/common/types/survey-item.type.ts

```diff
@@ -1,10 +1,18 @@
-export interface SingleChoice {
+interface SurveyQuestion {
     id: string;
     description: string;
+    type: SurveyItemType;
+    responses: string[];
 }
-export interface MultipleChoice {
-    id: string;
-    description: string;
+
+export interface SingleChoice extends SurveyQuestion {}
```

Comment:
```
export interface SingleChoice extends SurveyQuestion {
    type: SurveyItemType.SingleChoice;
}
```

Would make these awesome 🕺, will allow us to assert the interface based on `type`

Reply:
Of course, what was I thinking!

Reply:
I have no clue how to implement this. I'm stuck in typing hell.